src/Phinx/Config/Config.php

```diff
@@ -232,6 +232,14 @@ public function getAlias($alias)
         return !empty($this->values['aliases'][$alias]) ? $this->values['aliases'][$alias] : null;
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
Can we use the proper `@inheritDoc` tag here instead? This one means partial overwrite.

Reply:
Ah. My bad.

Reply:
Hmmm. I'm just following the existing pattern. If this is wrong, then a separate issue should be raised to fix this.